src/cryptonote_core/service_node_voting.cpp

```diff
@@ -58,10 +58,12 @@ namespace service_nodes
     return true;
   }
 
+  // TODO(loki): Post HF12 remove no more legacy votes should be propagated
```

Comment:
Won't they still be needed for blockchain syncing?

Reply:
The votes stored in the tx extra deregister legacy ain't the same as these deregister votes which are votes defined independently in the p2p layer.

Reply:
Oh, gotcha.